src/components/molecules/ArticleListTrashed/index.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="article-list">
+    <div v-if="errorMessage" class="article-list__notice--error">
+      <app-text bg-error>{{ errorMessage }}</app-text>
+    </div>
+    <app-heading :level="1">{{ articleTitle }}</app-heading>
```

Comment:
[imo]
articleTitleには「削除済み記事一覧」という文言が入ると思いますが、そもそもpropsで「削除済み記事」と言う文言を渡さなくても良いかなと思います！

app-headingに直接「削除済み記事一覧」を渡してあげる方がシンプルです。

```suggestion
    <app-heading :level="1">削除済み記事一覧</app-heading>
```

理由としては、コンポーネントの命名がArticleListTrashedとなっていることからこのコンポーネントは削除済み記事一覧以外で使われることはないと思うので、タイトルも固定してしまって良いため。

Reply:
修正しました！